oneanddone/base/middleware.py

```diff
@@ -16,7 +16,8 @@ def process_request(self, request):
         if request.user.is_authenticated():
             for attempt in request.user.attempts_requiring_notification:
                 messages.warning(request,
-                                 _('The task that you were working on, "%s", has expired or become invalid '
```

Comment:
With 99 chars per line this could go:

```python
messages.warning(request,  _('The task that you were working on, "%s", ' 
                                                  'has expired or become invalid '
                                                  'and therefore has been closed.' % attempt.task.name),
                                extra_tags='modal-message')
```